### PR TITLE
mysql-shell 8.3.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/m/mysql-shell.rb [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
   desc "Interactive JavaScript, Python or SQL interface"
   homepage "https://dev.mysql.com/downloads/shell/"
 
   livecheck do
     url "https://dev.mysql.com/downloads/shell/?tpl=platform&os=33"
-    regex(/mysql[._-]shell[._-]v?(\d+(?:\.\d+)+)[._-]macos13[._-]#{arch}\.dmg/i)
+    regex(/mysql[._-]shell[._-]v?(\d+(?:\.\d+)+)[._-]macos14[._-]#{arch}\.dmg/i)
   end
 
   depends_on macos: ">= :monterey"
```
Comment (bevanjkay) [inline Casks/m/mysql-shell.rb:18]:
```suggestion
  depends_on macos: ">= :ventura"
```

Screenshot 2024-01-25 at 11 03 28 am

**Review by bevanjkay — DISMISSED** [review]

File: Casks/m/mysql-shell.rb (hunk 1 above)
Comment (krehel) [reply Casks/m/mysql-shell.rb:18]: Can we keep the older version which appears to support both Big Sur and Monterey along with coverage for the latest version that covers Ventura and Sonoma?  That way we can preserve some previous support.

Comment (chenrui333) [reply Casks/m/mysql-shell.rb:18]: @krehel you mean keep the old 8.2.1, can you propose some change?

Comment (krehel) [reply Casks/m/mysql-shell.rb:18]: Yes. Sure, I can fiddle with it.

Comment (krehel) [reply Casks/m/mysql-shell.rb]: Pushed something to give this a try.

**Comment by krehel** [comment]
Like some extra eyes on this, please and thank you. It's not my favorite, but does seem to work ok.

**Review by p-linnane — COMMENTED** [review]
Looks good overall. Can we make the spacing above the livecheck blocks consistent though?

**Comment by krehel** [comment]
Yeah, I see now one was inconsistent.  I modified the rest to include a space, as that seems what we've done in other Casks.  `brew style` didn't seem to pick this up.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**